api/src/opentrons/protocol_engine/commands/load_liquid.py

```diff
@@ -0,0 +1,71 @@
+"""Load liquid command request, result, and implementation models."""
+from pydantic import BaseModel, Field
+from typing import Optional, Type, Dict
+from typing_extensions import Literal
+
+from .command import AbstractCommandImpl, BaseCommand, BaseCommandCreate
+
+
+LoadLiquidCommandType = Literal["loadLiquid"]
+
+
+class Liquid(BaseModel):
+    """Payload required to create a liquid."""
+
+    id: str
+    displayName: str
+    description: str
+    displayColor: Optional[str]
+
+
+class LoadLiquidParams(BaseModel):
+    """Payload required to load a liquid into a well."""
+
+    liquidId: str = Field(
+        ...,
+        description="Unique identifier generated by the app of instance of liquid to load.",
```

Comment:
```suggestion
        description="Unique identifier of the liquid to load.",
```

My understanding is that this identifier will be generated by either:

* Protocol Designer
* The Python Protocol API

And not the Opentrons App.